src/main/java/http/request/RequestHeader.java

```diff
@@ -0,0 +1,60 @@
+package http.request;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class RequestHeader {
+    private final Map<String, String> params;
+    private final Cookie cookie;
+
+    public RequestHeader(BufferedReader br) throws IOException {
+        params = new HashMap<>();
+        String line = br.readLine();
+        if (line.isEmpty()) {
+            throw new IllegalArgumentException("Line is empty.");
+        }
+        while ((line != null) && !line.isEmpty()) {
+            String[] token = line.split(": ");
+            if (token.length != 2) {
+                throw new IllegalArgumentException("No value for the key: " + token[0]);
+            }
+            params.put(token[0].toLowerCase(), token[1]);
+            line = br.readLine();
+        }
+        if (params.containsKey("cookie")) {
+            this.cookie = new Cookie(params.get("cookie"));
+        } else {
+            this.cookie = new Cookie("");
+        }
```

Comment:
```suggestion
        this.cookie = new Cookie(params.getOrDefault("cookie", ""));
```

getOrDefault를 활용할 수 있겠네요!